modules/apps/transfer/keeper/grpc_query.go

```diff
@@ -22,9 +24,16 @@ func (q Keeper) DenomTrace(c context.Context, req *types.QueryDenomTraceRequest)
 		return nil, status.Error(codes.InvalidArgument, "empty request")
 	}
 
-	hash, err := types.ParseHexHash(req.Hash)
+	var hash tmbytes.HexBytes
+	var err error
+	if strings.HasPrefix(req.Hash, "ibc/") {
+		hash, err = types.ParseHexHash(req.Hash[4:])
+	} else {
+		hash, err = types.ParseHexHash(req.Hash)
+	}
```

Comment:
Have you considered [`strings.TrimPrefix()`](https://pkg.go.dev/strings#TrimPrefix): 

```
hash, err := types.ParseHexHash(strings.TrimPrefix(req.Hash, "ibc/"))
if err != nil {
    // handle
}
```

Reply:
ohhh that is much more convenient, thank so much